doc/contrib/recording.md

```diff
@@ -0,0 +1,81 @@
+# Session Recording Technical Documentation
+
+After every `kdbSet` action, the changes are calculated using Elektra's powerful changetracking API.
+The result of the calculation is an `ElektraDiff` instance we'll call _part diff_ throughout this document.
```

Comment:
Please add to glossary, eventually we will need it at other places, too. Why is it **part** diff?

Reply:
>  Why is it part diff?

Because it's only a part of the whole session diff. I'm open for a better name if you have one.

Reply:
I've added that to the glossary now.